terraform/bootstrap/destroy_service_account.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+org="gsa-tts-benefits-studio"
```

Comment:
issue: Either remove the default or update to our org:

```suggestion
org="gsa-tts-devtools-prototyping"
```

Reply:
As discussed in standup, this is a placeholder, so you can ignore my comment!